src/MessageOutbox/RelayMessages.php

```diff
@@ -0,0 +1,9 @@
+<?php
+declare(strict_types=1);
+
+namespace EventSauce\MessageOutbox;
+
+interface RelayMessages
```

Comment:
I think this is a good moment to introduce the interface, but I can easily drop this last commit if you prefer not to have it.

Reply:
It think it makes sense to have this